src/Cards.cpp

```diff
@@ -0,0 +1,122 @@
+
+#include "Cards.h"
+#include <ctime>
+#include <algorithm>
+#include <iostream>
+
+Cards::Cards() {
+    this->good = nullptr;
+    this->action = nullptr;
+}
+
+Cards::~Cards() {
+}
+
+Cards::Cards(const string &good, const string &action) : good(good), action(action) {}
+
+const string &Cards::getGood() const {
+    return good;
+}
+
+void Cards::setGood(const string &good) {
+    Cards::good = good;
+}
+
+const string &Cards::getAction() const {
+    return action;
+}
+
+void Cards::setAction(const string &action) {
+    Cards::action = action;
+}
```

Comment:
Following same comments as below. getters need to return pointers. Setters need to set the pointers.